worker/status_worker.go

```diff
@@ -139,7 +139,8 @@ func (sw *statusWorker) processRun(run state.Run) {
 			if updatedRun.MaxMemoryUsed != run.MaxMemoryUsed ||
 				updatedRun.MaxCpuUsed != run.MaxCpuUsed ||
 				updatedRun.Cpu != run.Cpu ||
-				updatedRun.Memory != run.Memory {
+				updatedRun.Memory != run.Memory ||
+				updatedRun.PodEvents != run.PodEvents {
```

Comment:
out of curiosity, is the `!=` operator sufficient for comparing list of structs??